docs/db/datasources.md

```diff
@@ -10,9 +10,11 @@ Graphback provides support to multiple databases when performing migrations.
 
 Our migration engine is based on Knex.js and we actively supporting and testing following database systems:
 
-- PostgreSQL
-- MySQL 
-- SQLite (Limited ability to modify tables - requires recreation of the database for each change)
+- PostgreSQL (Using Knex.js)
+- MySQL (Using Knex.js) 
```

Comment:
Not tested so should this be removed as "supported and tested"?

Reply:
This entire thing will need to be updated. We have community member checking MySQL now

Reply:
this could be part o the documentation issue